manim/mobject/svg/code_mobject.py

```diff
@@ -229,6 +316,13 @@ def gen_html_string(self):
             file.close()
 
     def gen_code_json(self):
+        """Function to background_color, generate code_json and tab_spaces from html_string.
+        background_color is just background color of displayed code.
+        code_json is 2d array with rows as line numbers
+        and columns as a array with length 2 having text and text's color value.
+        tab_spaces is 2d array with rows as line numbers
+        and columns as corresponding number of indentation_chars in front of that line in code.
+        """
```

Comment:
Returns?

Reply:
If it makes sense for this function to not have a return statement, then let's leave it as is for now.